### PR TITLE
[WFLY-11479] Add documentation note about silent mode for HTTP BASIC auth

### PR DESCRIPTION
**Pull request by skyllarr**
Documentation note for new feature [WFLY-11479] HTTP Basic Authentication - Silent Operation. 

This documentation note is done for new feature regarding JIRA issues:
https://issues.jboss.org/browse/WFLY-11479
https://issues.jboss.org/browse/EAP7-1154
https://issues.jboss.org/browse/ELY-1466

### CODE REVIEW

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 1]
```diff
@@ -689,6 +689,14 @@ Example _web.xml_ with _BASIC_ Authentication
 </web-app>
 ----
 
+_BASIC_ Authentication can be configured to be silent
+[source, xml]
+----
+<auth-method>BASIC?silent=true</auth-method>
```
[the remark this answers is not in the document]
Comment (Skyllarr) [reply docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc]: I think Elytron reads properties set in login-config in web.xml with AuthenticationManager: https://github.com/wildfly-security/elytron-web/blob/master/undertow-servlet/src/main/java/org/wildfly/elytron/web/undertow/server/servlet/AuthenticationManager.java#L128 .

[the remark this answers is not in the document]
Comment (mchoma) [reply docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc]: Ah ok, I wasn't aware of it. Thanks.

**Comment by bstansberry** [comment]
@mchoma @fjuma ok with you?

**Review by mchoma — APPROVED** [review]

**Comment by mchoma** [comment]
ok

**Review by fjuma — APPROVED** [review]

**Pull request merged**